wiki/Article_Styling_Criteria/en.md

```diff
@@ -141,42 +174,86 @@ The osu!taiko Champion title goes to [\_yu68](https://osu.ppy.sh/users/6170507).
 
 The escaping character in Markdown is the backslash (`\`). To escape the escape character for a literal backslash, use `\\`.
 
-The title of an article must never be escaped; otherwise, the backslashes will show up on the website.
+The title of an article must never be escaped, as titles are parsed as raw text, not Markdown.
 
-### Emphasizing
+#### Whitespace
 
-*Note: Keep in mind that the misuse and abuse of emphasis will reduce its effectiveness!*
+One empty line between two bodies of text is parsed as two separate paragraphs. Adding more empty lines afterwards will not affect spacing of the paragraphs; it will be parsed as if they were one empty line.
```

Comment:
Should this be part of the `Line breaks` section? This is talking about lines.

Whitespaces are *unnecessary spaces* left in code and are invisible in previews.

Reply:
I'll change it to `Empty lines`, line breaks is supposed to talk about markdown's version of the `<br/>` tag, which is called a line break, but considering that it is under `Raw text` I'll just rename the section below to `Whitespace`.